projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-set-header.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="commentSummaryHeader">
+  <button *ngIf="showCommentSummary"
```

Comment:
this seems to suggest that the header is only shown if the commentsSumary is shown, is that true? I thought it's always visible

Reply:
i am not sure for now is what we have... when we add new features in then we address the better logic.. is just the flag on the input attribute

Reply:
ok